src/content/1.7/modules/creation/good_practices.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Good practices for developing modules
+menuTitle: Good practices
+---
+
+# Good practices for developing modules
+
+## DOs and DON'Ts
+
+### Do
+
+- Add a link to your documentation included in your ZIP File and to your support page on Addons in the module interface.
+- Follow the PSR-2 standard for modules destined for PS 1.6.1.0 +: http://doc.prestashop.com/display/PS16/Coding+Standards
+- Create your own tables!
+- Develop your module in English, then use PrestaShop translation system to translate your module.
+- Go through directories using code and variables like: `dirname(__FILE__).'/../../config/config.inc.php'`
+- Use PrestaShop variables like: `_PS_CONFIG_DIR_ . '/config.inc.php'`
+
+### Don't
+
+- Don't insert external links into your module code or module documentation.
+- Don't provide your personal contact details in your module or module documentation.
+- Don't send your customers to your own support/ticket management platform.
+- Don't use external ajax files to perform ajax tasks.
+- Don't go through directories using code and variables like: `dirname(__FILE__).'/../../config/config.inc.php'`
```

Comment:
In the DOs it recommended:
Go through directories using code and variables like: `dirname(__FILE__).'/../../config/config.inc.php'`

am i missing something?

Reply:
Good point, I hadn't noticed that!